micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -402,6 +402,16 @@ private void monitor(MeterRegistry registry, ForkJoinPool fj) {
             .description(
                     "An estimate of the number of worker threads that are not blocked waiting to join tasks or for other managed synchronization threads")
             .register(registry);
+
+        Gauge.builder(metricPrefix + "executor.parallelism", fj, ForkJoinPool::getParallelism)
+            .tags(tags)
+            .description("The targeted parallelism level of this pool")
+            .register(registry);
+
+        Gauge.builder(metricPrefix + "executor.pool.size", fj, ForkJoinPool::getPoolSize)
+            .tags(tags)
+            .description("The number of worker threads that have started but not yet terminated")
+            .register(registry);
```

Comment:
The unit should also be `BaseUnits.THREADS` to match the corresponding meter for ThreadPoolExecutor.